gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json

```diff
@@ -22,6 +22,9 @@
         }]
       }
     },
+    "required": [
+      "error_message"
```

Comment:
Why is the error message required? Shouldn't we have a default one?

Reply:
true